### PR TITLE
Feature/update heading text themes

### PR DESCRIPTION
**Pull request by gopienkonikita**
Unused global styles were removed, text and header component styles were moved to themes.

### CODE REVIEW

File: src/themes/base.js [hunk 1]
```diff
@@ -6,7 +6,20 @@ const Base = {
   fontFamily: "Open Sans, sans-serif, Arial",
 
   text: {
-    color: globalColors.black
+    color: globalColors.black,
+    fontWeight: 500,
```
Comment (andreysavihin) [inline src/themes/base.js]: Why 500? Normal text same as 400 ( https://www.w3.org/TR/2018/REC-css-fonts-3-20180920/ )

File: src/themes/dark.js [hunk 2]
```diff
@@ -6,7 +6,20 @@ const Dark = {
   fontFamily: "Open Sans, sans-serif, Arial",
 
   text: {
-    color: globalColors.text
+    color: globalColors.text,
+    fontWeight: 500,
```
Comment (andreysavihin) [inline src/themes/dark.js]: Normal text same as 400

File: src/themes/base.js (hunk 1 above)
Comment (gopienkonikita) [reply src/themes/base.js]: Fixed, set to default normal

File: src/themes/dark.js (hunk 2 above)
Comment (gopienkonikita) [reply src/themes/dark.js]: Fixed, set to default normal

**Pull request merged**